Update magicavoxel from 0.99.4.1 to 0.99.4.2

GabLeRoux opened the pull request:
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

GabLeRoux commented:
I went with the update path, but I was getting the following error at first:

```
==> Upgrading 1 outdated package:
magicavoxel 0.99.4 -> 0.99.4.1
==> Upgrading magicavoxel
==> Satisfying dependencies
==> Downloading https://github.com/ephtracy/ephtracy.github.io/releases/download/0.99.4/MagicaVoxel-0.99.4.1-alpha-macos.zip
Already downloaded: /Users/gableroux/Library/Caches/Homebrew/downloads/ba62a2580b06d6bb5d089ed65365af497b44f08b17385386a2f418ae4ff0e556--MagicaVoxel-0.99.4.1-alpha-macos.zip
==> Verifying SHA-256 checksum for Cask 'magicavoxel'.
==> Note: Running `brew update` may fix SHA-256 checksum errors.
==> Purging files for version 0.99.4.1 of Cask magicavoxel
Error: Checksum for Cask 'magicavoxel' does not match.
Expected: da875b367cb4a9808fcaf906d3c39321c3715b302c5c2c438571698ed1d66238
  Actual: 60617f1d57b02c1d95764a4130280b5a67dc0d4ed9c68c4dd09ee4e1c1721b2c
    File: /Users/gableroux/Library/Caches/Homebrew/downloads/ba62a2580b06d6bb5d089ed65365af497b44f08b17385386a2f418ae4ff0e556--MagicaVoxel-0.99.4.1-alpha-macos.zip
To retry an incomplete download, remove the file above.
If the issue persists, visit:
  https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md
```

I just gave `cask-repair` a try and that was a very good experience 👍

GabLeRoux commented:
I'm not sure what's breaking the checks in the CI, anyone here with more experience than me with homebrew ci checks would like to provide details?

github actions Are failing with
```
Error: Not a GitHub pull request or commit: 
##[error]Process completed with exit code 1.
```
https://github.com/Homebrew/homebrew-cask/pull/69170/checks?check_run_id=224129463

and travis is failing with a similar error from what I shared here:  
https://github.com/Homebrew/homebrew-cask/pull/69170#issuecomment-531667223

https://travis-ci.com/Homebrew/homebrew-cask/jobs/236166429

```
 - appcast at URL 'https://github.com/ephtracy/ephtracy.github.io/releases.atom' does not contain the version number: '0.99.4.2'
style
196.59s==> brew cask style magicavoxel ✔
install
3.70s==> brew cask install magicavoxel ✘
==> Satisfying dependencies
==> Downloading https://github.com/ephtracy/ephtracy.github.io/releases/download
Already downloaded: /Users/travis/Library/Caches/Homebrew/downloads/0246af91b2b29b94dee4fde952adac5b03f90e458e628486bbdafc9f69f6b01d--MagicaVoxel-0.99.4.2-alpha-macos.zip
==> Verifying SHA-256 checksum for Cask 'magicavoxel'.
==> Note: Running `brew update` may fix SHA-256 checksum errors.
Checksum for Cask 'magicavoxel' does not match.
Expected: 24a83440e7f71fa52d548d45195f2fa060d754c624bfff518dbd595ac7868de4
  Actual: 953ba9242c431eb9c4160accdcf70e8aa8c8609da037bef69248e5372caad7b8
    File: /Users/travis/Library/Caches/Homebrew/downloads/0246af91b2b29b94dee4fde952adac5b03f90e458e628486bbdafc9f69f6b01d--MagicaVoxel-0.99.4.2-alpha-macos.zip
To retry an incomplete download, remove the file above.
If the issue persists, visit:
  https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md
uninstall
1.20s==> brew cask uninstall magicavoxel ✔
Error: Build failed.
```

Maybe there's a bug with `cask-repair` which doesn't extract the right `SHA-256`.

I'll add a new commit to this branch using the one from the error log (`953ba9242c431eb9c4160accdcf70e8aa8c8609da037bef69248e5372caad7b8`).

vitorgalvao commented:
> Maybe there's a bug with `cask-repair` which doesn't extract the right `SHA-256`.

Unlikely. Different versions on different CDN sources or a custom `.curlrc` are the usual culprits.

GabLeRoux commented:
> Unlikely. Different versions on different CDN sources or a custom .curlrc are the usual culprits.

Indeed, makes more sens. 

I do not have a custom have a custom `~/.curlrc` on my system. I've added a new commit 3eeeb8f to use the value from the previous run instead, and it's still failing, now with a new sha-256. It's like the archive is different each time, different cdn source could be the reason here.

The download url from their website gives:  
https://github.com/ephtracy/ephtracy.github.io/releases/download/0.99.4/MagicaVoxel-0.99.4.2-alpha-macos.zip

I tried locally again and got same value as the runner now:  
`2ee6e63c45b4e3d7a99b185d48bb9c2b8e8140e88d106c1d86303243379c2729`

So I just added e6eed00 which is passing now, but will it still pass in 2 hours or tomorrow? 😱 

@vitorgalvao what should be done and how can we investigate this?

GabLeRoux commented:
Thanks vitorgalvao 🎉
